test/integration/serverless/snapshots/error-java

```diff
@@ -99,6 +99,54 @@
     ],
     "dogsketches": []
   },
+  {
+    "metric": "aws.lambda.enhanced.errors",
+    "distributions": null,
+    "tags": [
+      "account_id:425362996713",
+      "architecture:XXX",
+      "aws_account:425362996713",
+      "dd_extension_version:123",
+      "env:integration-tests-env",
+      "function_arn:arn:aws:lambda:eu-west-1:425362996713:function:integration-tests-extension-XXXXXX-error-java",
+      "functionname:integration-tests-extension-XXXXXX-error-java",
+      "memorysize:1024",
+      "region:eu-west-1",
+      "resource:integration-tests-extension-XXXXXX-error-java",
+      "runtime:java8.al2",
+      "service:integration-tests-service",
+      "taga:valuea",
+      "tagb:valueb",
+      "tagc:valuec",
+      "tagd:valued",
+      "version:integration-tests-version"
+    ],
+    "dogsketches": []
+  },
+  {
+    "metric": "aws.lambda.enhanced.errors",
+    "distributions": null,
+    "tags": [
+      "account_id:425362996713",
+      "architecture:XXX",
+      "aws_account:425362996713",
+      "dd_extension_version:123",
+      "env:integration-tests-env",
+      "function_arn:arn:aws:lambda:eu-west-1:425362996713:function:integration-tests-extension-XXXXXX-error-java",
+      "functionname:integration-tests-extension-XXXXXX-error-java",
+      "memorysize:1024",
+      "region:eu-west-1",
+      "resource:integration-tests-extension-XXXXXX-error-java",
+      "runtime:java8.al2",
+      "service:integration-tests-service",
+      "taga:valuea",
+      "tagb:valueb",
+      "tagc:valuec",
+      "tagd:valued",
+      "version:integration-tests-version"
+    ],
+    "dogsketches": []
+  },
```

Comment:
Now that the integration test is using universal instrumentation, we are seeing these error metrics being created.